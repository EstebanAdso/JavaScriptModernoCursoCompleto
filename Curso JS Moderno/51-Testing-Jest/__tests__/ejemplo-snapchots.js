const cliente = {
    nombre: 'Carlos Esteban',
    balance: 500,
    tipo: 'Premium'
}

describe('Testing al cliente', () =>{
    test('Es Esteban?', () =>{
        //creara una carpeta de snapshot con el objeto y probara todo el objeto
        expect(cliente).toMatchSnapshot();
    });

})