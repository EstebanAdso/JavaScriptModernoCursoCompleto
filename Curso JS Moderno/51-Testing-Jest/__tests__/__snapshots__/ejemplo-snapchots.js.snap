// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing al cliente Es Esteban? 1`] = `
{
  "balance": 500,
  "nombre": "Carlos Esteban",
  "tipo": "Premium",
}
`;
